QUICKSTART-VMWARE.md

```diff
@@ -0,0 +1,255 @@
+cluster-certificate = "${CLUSTER_CERTIFICATE:-<CLUSTER_CERTIFICATE>}"
+EOF
+```
+
+### Accessing your Bottlerocket guest via host containers
+For remote access to your running Bottlerocket VMs, you will need to add additional user data to enable host containers.
+The Bottlerocket VMware images don't include any host containers enabled by default.
+But don't worry!
+You can use our [admin](https://github.com/bottlerocket-os/bottlerocket-admin-container) and/or [control](https://github.com/bottlerocket-os/bottlerocket-control-container) containers, they just need to be configured first.
+
+#### Admin container
+If you would like to use the admin container, you will need to create some base64 encoded user data which will be passed to the container at runtime.
+Full details are covered in the [admin container documentation](https://github.com/bottlerocket-os/bottlerocket-admin-container#authenticating-with-the-admin-container).
+If we assume you have a public key at `${HOME}/.ssh/id_rsa.pub`, the below will add the correct user data to your `userdata.toml`.
+```
+PUBKEY="${HOME}/.ssh/id_rsa.pub"
+ADMIN_USER_DATA="$(echo '{"ssh":{"authorized-keys":["'"$(cat ${PUBKEY})"'"]}}' | base64 -w 0)"
+
+cat <<EOF >>userdata.toml
+[settings.host-containers.admin]
+enabled=true
+user-data = "${ADMIN_USER_DATA:-<ADMIN_USER_DATA>}"
+EOF
+```
+
+#### Control container
+Enabling the control container is very similar to the admin container; you will create some base64 encoded user data that will be passed to the container at runtime.
+This user data includes an activation ID and code retrieved from AWS SSM.
+Full details can be found in the [control container documentation](https://github.com/bottlerocket-os/bottlerocket-control-container#connecting-to-aws-systems-manager-ssm).
+
+You'll first need an AWS account, and AWS CLI installed.
+Then you'll create a service role in that account to [grant AWS STS trust to the AWS Systems Manager service](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-service-role.html).
+```
+cat <<EOF > ssmservice-trust.json
+{
+    "Version": "2012-10-17",
+    "Statement": {
+        "Effect": "Allow",
+        "Principal": {
+            "Service": "ssm.amazonaws.com"
+        },
+        "Action": "sts:AssumeRole"
+    }
+}
+EOF
+
+# Create the role using the above policy
+aws iam create-role \
+    --role-name SSMServiceRole \
+    --assume-role-policy-document file://ssmservice-trust.json
+
+# Attach the policy enabling the role to create session tokens
+aws iam attach-role-policy \
+    --role-name SSMServiceRole \
+    --policy-arn arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore  
+```
+
+Once the above is created, we can use the role to create an activation code and ID.
+```
+export SSM_ACTIVATION="$(aws ssm create-activation \
+  --default-instance-name MyVMwareVMs \
+  --description "Activation for VMware VMs" \
+  --iam-role SSMServiceRole \
+  --registration-limit 1 \
```

Comment:
Maybe set --registration-limit to 100 for this example. By default the limit is 1 so no one would specify this manually. It might also be a good idea to mention that names and descriptions are optional.